Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "lingua-franca"
+name = "lingo"
```

Comment:
I would leave it as lingua-franca because we still want to squat that name on crates.io ;D

Reply:
```suggestion
name = "lingua-franca"
```